src/board.c

```diff
@@ -160,29 +186,133 @@ void module_init(void) {
     break;
   }
 
-
 //	xTaskCreate( DJICanTask, (const char*)"DJI CAN",
 //	STACK_SIZE_MIN, NULL, THREAD_PRIO_HIGH, NULL );
 }
 
+// Initialize the semaphores used for intra-task communication
+void task_semaphores_init() {
+  vSemaphoreCreateBinary(onScreenDisplaySemaphore);
+  vSemaphoreCreateBinary(onMavlinkSemaphore);
+  vSemaphoreCreateBinary(onUAVTalkSemaphore);
+}
+
+/*
+void watchdog_init() {
```

Comment:
This is just half-done stuff I commented out. It can come all the way out. As for git storing the changes, I've been trying to squash all my changes on a per-PR basis, so the main repo wouldn't have it, just my local one.